### PR TITLE
submit commonmark

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

**Comment by ericmjl** [comment]
@jakirkham: configparser is not installing on CircleCI. Any ideas why this might be happening?

File: recipes/commonmark/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - cmark = CommonMark.cmark:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - future
```
Comment (jakirkham) [inline recipes/commonmark/meta.yaml]: Could we please drop `future` from `build`?

File: recipes/commonmark/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  commands:
+    - cmark --help
+
+  requires:
+    - flake8
+    - hypothesis
+
+about:
+  home: https://github.com/rtfd/CommonMark-py
+  license: BSD License
```
Comment (jakirkham) [inline recipes/commonmark/meta.yaml]: Version?

Comment (jakirkham) [inline recipes/commonmark/meta.yaml]: License file?

File: recipes/commonmark/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+{% set md5 = "93fbcc5359935ca82ebf9de8e4a94a73" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
```
Comment (jakirkham) [inline recipes/commonmark/meta.yaml]: sha256?

File: recipes/commonmark/meta.yaml (hunk 1 above)
Comment (ericmjl) [inline recipes/commonmark/meta.yaml]: Done!

File: recipes/commonmark/meta.yaml (hunk 3 above)
Comment (ericmjl) [inline recipes/commonmark/meta.yaml]: Done!

**Comment by ericmjl** [comment]
@jakirkham all tests passed! yay! :smile:

File: recipes/commonmark/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  imports:
+    - CommonMark
+    - CommonMark.render
+    - CommonMark.tests
+
+  commands:
+    - cmark --help
+
+  requires:
+    - flake8
```
Comment (patricksnape) [inline recipes/commonmark/meta.yaml]: The tests aren't actually run so do we need `flake8` and `hypothesis`?

Comment (ericmjl) [inline recipes/commonmark/meta.yaml]: It came up in the automatic conda skeleton-generated script, so I left them in. I can take them out though.

Comment (ericmjl) [inline recipes/commonmark/meta.yaml]: Removed!

**Pull request merged**